app/middleware.py

```diff
@@ -66,3 +66,16 @@ async def redirect_latest(request: Request, call_next):
     else:
         response = await call_next(request)
         return response
+
+
+async def no_cache_response_header(request: Request, call_next):
+    """This middleware adds a cache control response header.
+
+    By default, specify no-cache. Individual endpoints can override this
+    header.
+    """
+    response = await call_next(request)
+    if "Cache-Control" not in response.headers:
+        response.headers["Cache-Control"] = "no-cache"
```

Comment:
Oh, just took it literally and thought it meant don't cache it. Maybe not, because we anyway don't use etags or anything to validate against, It seems like maybe we want to use no-store or max-age=0?

Reply:
In tile cache here, why did you use no-cache for those particular endpoints? Would the same apply here? 
https://github.com/wri/gfw-tile-cache/blob/production/app/middleware.py

Reply:
Probably, I think I didn't pay much attention to this detail either back then

Reply:
In general it would be fine to cache those responses for a couple of seconds as they do not change that often.

Reply:
Oh ok, I just changed it to be consistent with tile cache. Now it's max-age=0 by default, and no-cache for doc endpoints. We can just adjust it as needed.